great_expectations/rule_based_profiler/types/data_assistant_result/volume_data_assistant_result.py

```diff
@@ -16,10 +16,15 @@
 class VolumeDataAssistantResult(DataAssistantResult):
     def plot(
         self,
-        prescriptive: bool = False,
+        prescriptive: Optional[bool] = False,
```

Comment:
Why is the nullable now?

Reply:
It was wrong before. If you don't pass the parameter it defaults to False.

Reply:
But can it ever be `None`? If not, I don't think `Optional` fits here.

`Optional` denotes the possibility of a null value and confusingly has nothing to do with whether or not an argument is required/has a default value.

Reply:
Well that's news to me!

Reply:
✅